solutions/standard/variables.tf

```diff
@@ -352,19 +352,25 @@ variable "admin_pass_sm_secret_name" {
 ##############################################################
 
 variable "existing_code_engine_project_id" {
-  description = "Existing code engine project ID to deploy Kibana. If no value is passed, a new code engine project will be created."
   type        = string
+  description = "Existing code engine project ID to deploy Kibana. If no value is passed, a new code engine project will be created."
```

Comment:
`new code engine project will be created.` -> `new code engine project ID will be created`

Reply:
Done.